src/app/web-app.component.ts

```diff
@@ -113,6 +126,141 @@ export class WebAppComponent implements OnInit {
+          event.preventDefault();
+        },
+
+        // Create Center
+        'Control.Shift.q': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Frequent Postings
+        'Control.Shift.f': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting/journal-entries/frequent-postings']);
+          event.preventDefault();
+        },
+
+        // Closure Entries
+        'Control.Shift.e': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Journal Entries
+        'Control.Shift.j': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting/journal-entries']);
+          event.preventDefault();
+        },
+
+        // Reports
+        'Control.Shift.r': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Accounting
+        'Control.Shift.a': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting']);
+          event.preventDefault();
+        },
+
+        // Save/Submit Forms
+        'Control.s': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Run Report
+        'Control.r': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Cancel
+        'Control.Shift.x': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Logout
+        'Control.Shift.l': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Search
+        'Control.Shift.h': (event: KeyboardEvent): void => {
+          this.router.navigate(['help']);
+          event.preventDefault();
+        },
+
+        // Pagination: Next
+        'Control.n': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Pagination: Previous
+        'Control.p': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        'Alt.x': (event: KeyboardEvent): void => {
```

Comment:
This shortcut is to shift the focus on the search field in the toolbar above. (Please see community app for reference) The text field exists, figure out a way to make this work.

Reply:
I've figured out a way to do this, i need to call a method of another component from one component, i can do this by creeating a service, so am i allowed to create a service on my own?

Reply:
It depends on the work. There is no such restriction, you may create it. Though I'll let you know if it was unnecessary when you send the PR for review.

Reply:
Done.